scripts/kernel_bump.sh

```diff
@@ -0,0 +1,225 @@
+#!/bin/sh
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright (C) 2024 Olliver Schinagl <oliver@schinagl.nl>
+
+set -eux
+if [ -n "${DEBUG_TRACE_SH:-}" ] && \
+   [ "${DEBUG_TRACE_SH:-}" != "${DEBUG_TRACE_SH#*"$(basename "${0}")"*}" ] || \
+   [ "${DEBUG_TRACE_SH:-}" = 'all' ]; then
+	set -x
+fi
+
+REQUIRED_COMMANDS='
+	[
```

Comment:
Is this standalone opening bracket intentional?

Reply:
Should be! It's like `test`.

Reply:
Absolutely! I use this template for my scripts almost always, and I like to have a 'self-check' before doing things. Especially where late failure can be nasty.